### ISSUE TITLE
Fine-tuning stable-diffusion-xl-1024-v1-0 ModuleNotFoundError: No module named 'stability_sdk.finetune'

### ISSUE THREAD

**Issue by abdul7235** [Stability-AI/stability-sdk#297]
Hello, I am following the following document for finetuning stable-diffusion-xl-1024-v1-0 

https://next.platform.stability.ai/docs/features/fine-tuning

in the imports

from stability_sdk.finetune import (
    create_model, delete_model, get_model, list_models, resubmit_model, update_model,
    FineTuneMode, FineTuneParameters, FineTuneStatus
)
I get the error:

ModuleNotFoundError: No module named 'stability_sdk.finetune'

Can anyone please help me with the fix?

**Comment by andreybleme** [comment]
Have you figured that out @abdul7235 ?

Facing the exact same issue here.

**Comment by abdul7235** [comment]
No, I could not figure it out. I switeched to replicate. They provide a variety of generative models.

Here is the link: https://replicate.com/
